pkg/util/runtime/features.go

```diff
@@ -34,6 +34,9 @@ const (
 	// FeaturePlayerTracking is a feature flag to enable/disable player tracking features.
 	FeaturePlayerTracking Feature = "PlayerTracking"
 
+	// FeatureMTLSEnabled is a feature flag to enable/disable mTLS in the allocator.
+	FeatureMTLSEnabled Feature = "MTLSEnabled"
```

Comment:
New feature gates need to be documented at https://github.com/googleforgames/agones/blob/master/site/content/en/docs/Guides/feature-stages.md

Reply:
Done.